pkg/services/object/get/util.go

```diff
@@ -296,3 +262,31 @@ func (h *hasherWrapper) WriteChunk(p []byte) error {
 func prettyRange(rng *object.Range) string {
 	return fmt.Sprintf("[%d:%d]", rng.GetOffset(), rng.GetLength())
 }
+
+// DirectChildWriter streams child object payload directly into destination ChunkWriter
+// while capturing the header.
+type DirectChildWriter struct {
```

Comment:
```suggestion
type directChildWriter struct {
```
same for `NewDirectChildWriter`